source/2019-11-22-the-ember-times-issue-125.md

```diff
@@ -24,13 +24,11 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Reminder: EmberConf 2020 CFP, Due December 1 🎤](https://cfp.emberconf.com/events/emberconf-2020/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Each year, [EmberConf](https://emberconf.com/) gets to feature wonderful, *ambitious* talks and workshops. We encourage you to take stage in 2020 and [share what you learned (or want to learn) with the world](https://cfp.emberconf.com/events/emberconf-2020/)! You can give a 5, 15, or 30-min talk, or a 3-hour workshop. You can also submit more than 1 proposal.
```

Comment:
```suggestion
Each year, [EmberConf](https://emberconf.com/) gets to feature wonderful, *ambitious* talks and workshops. We encourage you to take to the stage in 2020 and [share what you learned (or want to learn) with the world](https://cfp.emberconf.com/events/emberconf-2020/)! You can give a 5, 15, or 30-min talk, or a 3-hour workshop. You can also submit more than 1 proposal.
```

Reply:
Thanks! It seems the saying is "take the stage," so let me change to that.